roles/nginx_hardening/defaults/main.yml

```diff
@@ -23,7 +23,6 @@ nginx_add_header:
 
 nginx_set_cookie_flag: "* HttpOnly secure"
 nginx_ssl_prefer_server_ciphers: "on"
-nginx_ssl_protocols: "TLSv1.2 TLSv1.3"
```

Comment:
I think, we should reverse this split, when we drop Debian 9 support. Add a Issue for future reference?

Reply:
https://github.com/dev-sec/ansible-collection-hardening/issues/528